CMakeLists.txt

```diff
@@ -14,13 +14,13 @@
 cmake_minimum_required(VERSION 3.13)
 
 project(re
-  VERSION 3.2.0
+  VERSION 3.3.0
   LANGUAGES C
   HOMEPAGE_URL https://github.com/baresip/re
   DESCRIPTION "Generic library for real-time communications"
 )
 
-set(PROJECT_SOVERSION 16) # bump if ABI breaks
+set(PROJECT_SOVERSION 17) # bump if ABI breaks
```

Comment:
I think this release needs no ABI bump, since we only added new functions with no other changes at first look:

```
Function symbols changes summary: 0 Removed, 5 Added function symbols not referenced by debug info
```